CLOUDP-128165: Create command "atlas backups schedule describe" (#1360)
mongodb/mongodb-atlas-cli: docs/atlascli/command/atlas-backups-schedule.txt

.. _atlas-backups-schedule:

======================
atlas backups schedule
======================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> Get cloud backup schedule of a given cluster in your project.

Reviewer: ```suggestion
Return a cloud backup schedule for the cluster you specify.
```

Options
-------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - -h, --help
     - 
     - false
     - help for schedule

Inherited Options
-----------------

.. list-table::
   :header-rows: 1
   :widths: 20 10 10 60

   * - Name
     - Type
     - Required
     - Description
   * - -P, --profile
     - string
     - false
     - Profile to use from your configuration file.

Related Commands
----------------

* :ref:`atlas-backups-schedule-describe` - Describe a cloud backup schedule of a given cluster.

.. toctree::
   :titlesonly:

   describe </command/atlas-backups-schedule-describe>

> 

Reviewer: Can you please add an example for this command? Thank you!

Author: I think there already is an example:
```
The following example describes the cloud backup schedule for the cluster Cluster0:
$ atlas backup schedule describe Cluster0
```
